config/jobs/image-pushing/k8s-staging-kwok.yaml

```diff
@@ -0,0 +1,26 @@
+postsubmits:
+  # this is the github repo we'll build from; this block needs to be repeated for each repo.
+  kubernetes-sigs/kwok:
+    - name: post-kwok-push-images
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        # this is the name of some testgrid dashboard to report to.
+        testgrid-dashboards: sig-scheduling, sig-k8s-infra-gcb
+      decorate: true
+      branches:
+        # Build tags also
+        # this is a regex for semver, from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+        - ^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
+      spec:
+        serviceAccountName: gcb-builder
+        containers:
+          - image: gcr.io/k8s-staging-test-infra/image-builder:v20220428-ae431ed1aa
+            command:
+              - /run.sh
+            args:
+              # this is the project GCB will run in, which is the same as the GCR images are pushed to.
+              - --project=k8s-staging-kwok
+              - --scratch-bucket=gs://k8s-staging-kwok-gcb
+              - --env-passthrough=PULL_BASE_REF
+              - --with-git-dir
```

Comment:
Is this parameter mandatory?

Reply:
That `--with-git-dir` and `--env-passthrough` are not mandatory

`--with-git-dir`
https://github.com/kubernetes/test-infra/blob/1576e68c95ec61ed19a1f3e8934cf5b990c39d37/images/builder/main.go#L140-L142
`--env-passthrough` 
https://github.com/kubernetes/test-infra/blob/1576e68c95ec61ed19a1f3e8934cf5b990c39d37/images/builder/main.go#L158-L168
`--scratch-bucket`
https://github.com/kubernetes/test-infra/blob/1576e68c95ec61ed19a1f3e8934cf5b990c39d37/images/builder/main.go#L188-L191
`--project`
https://github.com/kubernetes/test-infra/blob/1576e68c95ec61ed19a1f3e8934cf5b990c39d37/images/builder/main.go#L184-L186